README.md

```diff
@@ -139,6 +139,10 @@ These flags help catch three common issues in Kyo applications:
 
 In Kyo, computations are expressed via the infix type `<`, known as "Pending". It takes two type parameters:
 
+```scala 
```

Comment:
Kyo's readme code is validated by the CI, you'll need to add a mod here to ignore the code since it isn't complete to compile. We'd need to include internal details to make it complete, which doesn't make sense at this point of the docs

```suggestion
```scala mdoc:skip
```